src/components/payment-options.js

```diff
@@ -67,18 +67,26 @@ var StripeButton = React.createClass({
     name: React.PropTypes.string.isRequired,
     currency: React.PropTypes.object.isRequired
   },
-  onChange: function() {
+  onClick: function() {
     if (this.props.onClick) {
       this.props.onClick();
     }
-    setTimeout(() => {
-      this.props.onSubmit(this.props.validate, this.props.submit);
-    });
+    this.props.onSubmit(this.props.validate, this.props.submit);
     reactGA.event({
       category: "User Flow",
       action: "Stripe Clicked"
     });
   },
+  componentDidMount: function() {
+    // Need this because Chrome on iOS plus StripeCheckout triggers a popup.
+    // We need to ensure the popup isn't blocked.
+    this.input.addEventListener("click", this.onClick);
+  },
+  componentWillUnmount: function() {
+    // Need to ensure we remove it, or else we'll have a memory leak.
+    // This is why you don't use DOM events in React.
+    this.input.removeEventListener("click", this.onClick);
```

Comment:
If react ever tears down this component, we need to ensure we also remove things we added.